src/main/java/de/uni_tuebingen/ub/nppm/model/SelektionArealTyp.java

```diff
@@ -0,0 +1,27 @@
+package de.uni_tuebingen.ub.nppm.model;
+
+import javax.persistence.*;
+import java.util.*;
+
+@Entity
+@Table(name = "selektion_arealtyp")
+public class SelektionArealTyp extends SelektionBezeichnung {
+    @ManyToMany(mappedBy = "arealTyp")
+    private Set<Einzelbeleg> einzelbeleg = new HashSet<>();
+
+    public void addEinzelbeleg(Einzelbeleg beleg) {
+        this.getEinzelbeleg().add(beleg);
+    }
+
+    public void removeEinzelbeleg(int id) {
+        this.getEinzelbeleg().removeIf(e -> e.getId() == id);
+    }
+
+    public Set<Einzelbeleg> getEinzelbeleg() {
```

Comment:
Basically OK, but as also discussed with @rnaiser regarding other PRs:
- If a function returns a Set, it should be called by the plural form => getEinzelbelege
- It almost never makes sense to have a setEinzelbelege function called with a set. Instead it's always better to have single addEinzelbeleg and removeEinzelbeleg functions.

Reply:
fixed!